vscode-css/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "vscode-css-languageserver",
+  "description": "CSS/LESS/SCSS language server",
+  "version": "1.0.0",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "engines": {
+    "node": "*"
+  },
+  "main": "./out/cssServerMain",
+  "dependencies": {
+    "typescript": "^3.8.3",
```

Comment:
In this case it should actually not be needed in production. :)
(Haven't verified myself, but didn't see reference in any compiled file)

Reply:
yeah. i didn't find it in `out/` either. (via ripgrep)